### DISCUSSION TITLE
[Node Operator Question]  Ancient blocks frozen already

### DISCUSSION THREAD

**Discussion by calintatar1** [ethereum-optimism/developers#593] [Node Operators]
### Are you running the most up to date node software?

- [X] Yes

### Did you check the documentation?

- [X] Yes

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

 Ancient blocks frozen already            threshold=117,387,667 frozen=117,387,668

waited already more than 2 days ,but nothing happen

op-geth params:
--ws \
  --ws.port=9993 \
  --ws.addr=0.0.0.0 \
  --ws.origins="*" \
  --ws.api=eth,net,debug \
  --http \
  --discovery.port=30304 \
  --port=30304 \
  --nat=none \
  --op-network=op-mainnet \
  --http.port=8547 \
  --rollup.halt=major \
  --http.addr=0.0.0.0 \
  --http.vhosts="*" \
  --http.corsdomain="*" \
  --http.api=eth,net,debug \
  --authrpc.addr=localhost \
  --authrpc.jwtsecret=/home/optimism/optimism/jwt.txt \
  --authrpc.port=8751 \
  --authrpc.vhosts="*" \
  --datadir=/home/optimism/geth-data \
  --verbosity=3 \
  --rollup.disabletxpoolgossip=true \
  --rollup.sequencerhttp=https://mainnet-sequencer.optimism.io \
  --nodiscover \
  --syncmode=full \
  --maxpeers=0 \
  --verbosity=4 \
  --metrics \
  --metrics.addr=0.0.0.0 \
  --metrics.port=6770 \
  --networkid=10 \
  --pprof \
  --pprof.addr=0.0.0.0 \
  --pprof.port=6760 \
  --gcmode=archive \
  --snapshot=false

op-node params:

 --l1= \
  --l2=http://127.0.0.1:8751 \
  --l1.beacon= \
  --network=op-mainnet \
  --p2p.listen.tcp=9223 \
  --rpc.addr=0.0.0.0 \
  --rpc.port=8845 \
  --l2.jwt-secret=/home/optimism/optimism/jwt.txt \

### Protocol Description

geth version 1.101408.0-stable-5c2e7586
op-node version v1.9.1-4797ddb7-1724853807

archive mainnet node

### Node Logs

 Ancient blocks frozen already            threshold=117,387,667 frozen=117,387,668

**Comment by hash1go** [comment]
Hello. Could you please describe in more detail what you will not be resolving?

**Reply by calintatar1** [reply]
The node remains stuck on the same block, showing the same message. The block count is not increasing."

Let me know if you'd like further adjustments

**Reply by hash1go** [reply]
Please share the logs. Also, did the block synchronization stop suddenly?
Additionally, why did you set maxpeers=0?
https://docs.optimism.io/builders/node-operators/configuration/execution-config#maxpeers

Also, it seems that the L1 RPC settings have not been configured.
https://docs.optimism.io/builders/node-operators/tutorials/mainnet#start-op-node
https://docs.optimism.io/builders/node-operators/configuration/consensus-config#l1
https://docs.optimism.io/builders/node-operators/configuration/consensus-config#l1beacon

I recommend carefully reading the documentation again to check if there are any other issues with your configuration or setup.

**Reply by calintatar1** [reply]
he block synchronization stop suddenly, yes i tried to change maxpeers=50 but no improvment.
L1 is configured but i didn't add the endpoint here ,i have the same setup on other server and it's working ,so trust me i know any step from documentation

**Reply by hash1go** [reply]
Is the port setting of 30304 because you have multiple nodes? The default is 30303.
With 30304, there could be cases where it becomes difficult to discover other nodes.

Also, please share the logs.

**Reply by calintatar1** [reply]
I solved the problem by downloading a new snapshot. A helpful trick is to start the op-node first and then restart op-geth. This makes the frozen ancient block logs disappear.
